google-cloud-examples/src/test/java/com/google/cloud/examples/translate/snippets/ITTranslateSnippetsbeta.java

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.examples.translate.snippets;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import com.google.api.gax.paging.Page;
+import com.google.cloud.storage.Blob;
+import com.google.cloud.storage.Storage;
+import com.google.cloud.storage.Storage.BlobListOption;
+import com.google.cloud.storage.StorageOptions;
+import com.google.cloud.translate.v3beta1.BatchTranslateResponse;
+import com.google.cloud.translate.v3beta1.DeleteGlossaryResponse;
+import com.google.cloud.translate.v3beta1.Glossary;
+import com.google.cloud.translate.v3beta1.TranslateTextResponse;
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runners.MethodSorters;
+
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+public class ITTranslateSnippetsbeta {
+
+  private static final String projectId = System.getenv("PROJECT_ID");
+  private static final String glossaryName = "glossary-testing";
+
+  private ByteArrayOutputStream bout;
+  private PrintStream out;
+
+  @Before
+  public void setUp() {
+    bout = new ByteArrayOutputStream();
+    out = new PrintStream(bout);
+    System.setOut(out);
+  }
+
+  @After
+  public void tearDown() {
+    System.setOut(null);
+  }
+
+  @Test
+  public void test1_testListSupportedLanguages() {
```

Comment:
please remove all "test_" prefixes

Reply:
I have put that in so we can have the tests run in that particular order. First test_1 , test_2 and finally test_3. Please let me know if there is any other way in which I could refactor this.